Gds/src/fprime_gds/flask/static/js/vue-support/event.js

```diff
@@ -75,15 +75,18 @@ Vue.component("event-list", {
             // Remap command EVRs to expand opcode for visualization pruposes
             let groups = null
             if (item.template.severity.value == "EventSeverity.COMMAND" && (groups = display_text.match(OPREG)) != null) {
+                let component = "UNKNOWN";
                 let mnemonic = "UNKNOWN";
                 let id = parseInt(groups[1]);
                 for (let command in this.commands) {
                     command = this.commands[command];
                     if (command.id == id) {
+                        component = command.comp_name;
```

Comment:
Try using "command.full_name", it should already be comp.command and is what we used to execute commands.

Reply:
Fixed this in [38f4f7e](https://github.com/nasa/fprime/pull/346/commits/38f4f7e42f647624faa18b3525cea17887c4c0b9)